packages/visual-editor/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@yext/visual-editor",
   "description": "Component library for Yext Pages Visual Editor",
-  "version": "0.0.39",
+  "version": "0.0.40",
```

Comment:
You shouldn't update the package version in a PR since this isn't cutting a new npm version.